### PR TITLE
maintenance page aip

### PR DESCRIPTION
**Pull request by lewis150193**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/secure/RapidBoard.jspa?rapidView=1190&projectKey=RIA&view=detail&selectedIssue=RIA-2635

### Change description ###
Maintenance page AIP

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # https://help.github.com/en/articles/about-code-owners
 
-* @hmcts/platform-engineering @hmcts/devops
+* @hmcts/platform-engineering @hmcts/devops @hmcts/ia-aip-frontend
```
Comment (timja) [inline CODEOWNERS]:
```suggestion
* @hmcts/platform-engineering @hmcts/devops
```

File: iaaip/index.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    <div class="govuk-header__logo">
+        <span class="govuk-header__logotype">
+          <img src="/assets/images/hmcts-logo-compress.svg" alt="HM Courts and Tribunals Service">
+        </span>
+    </div>
+    <div class="govuk-header__content govuk-header__content--flex">
+      <a href="/start-appeal" class="govuk-header__link govuk-header__link--service-name">
+        Appeal an immigration or asylum decision
+      </a>
+
```
Comment (chrisgrimble) [inline iaaip/index.html]: Should we have a link to the dead service?

File: iaaip/index.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+      <a href="/logout" class="govuk-header__link govuk-header__link--service-name govuk-header__link--signout">
+        Sign out
+      </a>
```
Comment (chrisgrimble) [inline iaaip/index.html]: Don't think we want a sign out link on here.

File: iaaip/index.html [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+          <li class="govuk-footer__inline-list-item">
+            <a class="govuk-footer__link" href="https://hmcts-access.service.gov.uk/contact-us">
+              Contact Us
+            </a>
+          </li>
+
+          <li class="govuk-footer__inline-list-item">
+            <a class="govuk-footer__link" href="/cookie-policy">
+              Cookies
```
Comment (chrisgrimble) [inline iaaip/index.html]: Don't think we should have any of these links in the footer as they will be dead and redirect back to this page.

File: iaaip/index.html [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+      </div>
+    </div>
+  </div>
+</footer>
+
+
+
+
+
+<script src="/all.js"></script>
```
Comment (chrisgrimble) [inline iaaip/index.html]: Why do we need javascript for this page?

File: iaaip/index.html [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+
+<a href="#main-content" class="govuk-skip-link">Skip to main content</a>
+
+
+
+
+<div id="cookie-banner" style="display: none;">
+  <div class="govuk-width-container">
+    <p>GOV.UK uses cookies to make the site simpler. <a href="/cookie-policy" class="govuk-link" title="Find out more about cookies">Find out more about cookies</a></p>
```
Comment (chrisgrimble) [inline iaaip/index.html]: Thins link won't work as we are shuttered

File: iaaip/index.html [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+  <title>
+    Start your appeal - Appeal an immigration or asylum decision - HMCTS
+  </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+  <meta name="theme-color" content="#0b0c0c">
+
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+
+  <link rel="shortcut icon" href="/assets/images/favicon.ico" type="image/x-icon">
```
Comment (chrisgrimble) [inline iaaip/index.html]: Has this been uploaded to somewhere the shutter page can see.

File: iaaip/index.html [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+
+
+
+  <base href="https://localhost:3000">
```
Comment (chrisgrimble) [inline iaaip/index.html]: Get rid of this

File: iaaip/main.js [hunk 9]
```diff
@@ -0,0 +1 @@
...maica"},{value:"JP",name:"Japan"},{value:"JE",name:"Jersey"},{value:"JO",name:"Jordan"},{value:"KZ",name:"Kazakhstan"},{value:"KE",name:"Kenya"},{value:"KI",name:"Kiribati"},{value:"KP",name:"Korea, Democratic People's Republic of"},{value:"KR",name:"Korea, Republic of"},{value:"KW",name:"Kuwait"},{value:"KG",name:"Kyrgyzstan"},{value:"LA",name:"Lao PDR"},{value:"LV",name:"Latvia"},{value:"LB",name:"Lebanon"},{value:"LS",name:"Lesotho"},{value:"LR",name:"Liberia"},{value:"LY",name:"Libya"},{value:"LI",name:"Liechtenstein"},{value:"LT",name:"Lithuania"},{value:"LU",name:"Luxembourg"},{value:"MK",name:"Macedonia, Republic of"},{value:"MG",name:"Madagascar"},{value:"MW",name:"Malawi"},{value:"MY",name:"Malaysia"},{value:"MV",name:"Maldives"},{value:"ML",name:"Mali"},{value:"MT",name:"Malta"},{value:"MH",name:"Marshall Islands"},{value:"MQ",name:"Martinique"},{value:"MR",name:"Mauritania"},{value:"MU",name:"Mauritius"},{value:"YT",name:"Mayotte"},{value:"MX",name:"Mexico"},{value:"FM",name:"Micronesia, Federated States of"},{value:"MD",name:"Moldova"},{value:"MC",name:"Monaco"},{value:"MN",name:"Mongolia"},{value:"ME",name:"Montenegro"},{value:"MS",name:"Montserrat"},{value:"MA",name:"
```
Comment (chrisgrimble) [inline iaaip/main.js]: Is this file used?

File: iaaip/index.html (hunk 8 above)
Comment (timja) [reply iaaip/index.html]: this is what's causing the rendering to be broken for me

File: iaaip/index.html [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+  <link href="/main.css" rel="stylesheet" type="text/css">
```
Comment (timja) [inline iaaip/index.html]: why aren't you using the govuk-frontend css file?
https://github.com/hmcts/azure-shutter-pages/blob/master/default/govuk-frontend-ie8-3.4.0.min.css

https://github.com/hmcts/azure-shutter-pages/blob/master/default/index.html

Was this based off the default shutter page or the existing one?

File: iaaip/index.html [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
....204-.78.085-.441.011-.984-.983-1.64.508.125.808.583 1.198.232-.35-.011-.356-.774-1.622-1.407.48-.011.859.769 1.418.328-.525-.062-.7-.995-1.65-1.289.328.04.667.198.95-.187-.328.08-.52-.226-.877-.254.13-.412.102-.661-.04-.966-.248.017-.502.09-.683.243-.028-.226-.266-.238-.475-.22.011.095.057.237-.05.26-.17.033-.735.186-1.17.35-.436.17-.933.48-1.08.576-.09.062-.164-.079-.22-.147-.153.13-.306.3-.193.47-.243.022-.509.084-.735.265 0 .283.232.554.583.707-.588.582-.057 1.119-.571 1.627.723.029.599-.983.966-1.124-.04.712-.435.836-.435 1.6 0 .406.243.853.028 1.124.605-.164.447-1.51.825-1.91.068.231.029.452-.05.683-.215.628.022 1.481-.306 1.758.616-.062.69-1.051 1.204-1.418-.435.893-.887 2.797-1.537 3.753-1.057 1.424-1.679 3.16-2.645 3.16-.43 0-.752-.26-.752-.803 0-1.617 2.775-2.73 2.775-4.454 0-.77-.627-1.063-1.017-1.063-.266 0-.582.13-.848.35a.963.963 0 0 0-.328-.073c-.983 0-.718 1.763-1.577 1.746.34.193.921-.18 1.266-.672-.141 1.142-1.792.842-1.662 2.216.232-1.074 2.855-.922 2.855-2.55 0-.186-.096-.339-.226-.457.198-.125.384-.176.554-.176.316 0 .599.238.599.696 0 1.447-2.758 2.662-2.758 4.437 0 .3.09.548.231.746-.192.084-.395.045-.56-.153-.016.458.64.746 1.046.514.09.034.187.051.289.057-.
```
Comment (timja) [inline iaaip/index.html]: is this relevant? I can't even see your app on there

Full screenshot:
image

**Pull request merged**